internal/manager/task_transcode.go

```diff
@@ -111,14 +108,14 @@ func (t *GenerateTranscodeTask) isTranscodeNeeded() bool {
 	container := ""
 	audioCodec := ffmpeg.MissingUnsupported
 	if t.Scene.AudioCodec.Valid {
-		audioCodec = ffmpeg.AudioCodec(t.Scene.AudioCodec.String)
+		audioCodec = ffmpeg.ProbeAudioCodec(t.Scene.AudioCodec.String)
 	}
 
 	if t.Scene.Format.Valid {
 		container = t.Scene.Format.String
 	}
 
-	if ffmpeg.IsStreamable(videoCodec, audioCodec, ffmpeg.Container(container)) {
+	if ffmpeg.IsStreamable(videoCodec, audioCodec, ffmpeg.Container(container)) != nil {
```

Comment:
If file `is` streamable no transcode is needed
```suggestion
	if ffmpeg.IsStreamable(videoCodec, audioCodec, ffmpeg.Container(container)) == nil {
```